packages/expo/src/singleton.ts

```diff
@@ -41,6 +49,24 @@ export function buildClerk({ key, tokenCache }: BuildClerkOptions): HeadlessBrow
 
       const jwt = await getToken(KEY);
       (requestInit.headers as Headers).set('authorization', jwt || '');
+
+      // indicate the device; non-identifying information
+      (requestInit.headers as Headers).set('x-expo-device-brand', Device.brand ?? NULL_VALUE);
+
+      // [isDevice]: true if the app is running on a real device and false if running in a simulator or emulator. On web, this is always set to true.
+      (requestInit.headers as Headers).set('x-expo-device-is-device', Device.isDevice ? 'true' : 'false');
+      (requestInit.headers as Headers).set('x-expo-device-brand', Device.brand ?? NULL_VALUE);
+      (requestInit.headers as Headers).set('x-expo-device-model-id', Device.modelId ?? NULL_VALUE);
+      (requestInit.headers as Headers).set('x-expo-application-id', Application.applicationId ?? NULL_VALUE);
+      (requestInit.headers as Headers).set('x-expo-application-name', Application.applicationName ?? NULL_VALUE);
+      (requestInit.headers as Headers).set(
+        'x-expo-native-application-version',
+        Application.nativeApplicationVersion ?? NULL_VALUE,
+      );
```

Comment:
What insight might we gain from the app name / id?

Reply:
💭 app name == `Expo Go` is roughly equivalent to request.origin == `http://localhost:3000`

Reply:
More generally, my thinking here falls mostly under "I don't know what I don't know" regarding insights. But presence of this data should at least enable gaining any insight at all!